scripts/add_content_to_db.py

```diff
@@ -71,30 +71,37 @@
 
     with open(args.csv, mode="r", encoding="utf-8") as file:
         reader = csv.DictReader(file)
-        tags = [val for row in reader for val in row["tags"].strip().split(" ")]
-        tags = set(tags)
-        tags_map = {}
+        tags = [val.upper() for row in reader for val in eval(row["tags"])]
+        tags_db = requests.get(tags_endpoint, headers=headers)
+        tags_map = {val["tag_name"]: val["tag_id"] for val in tags_db.json()}
+
+        tags_db = [val["tag_name"] for val in tags_db.json()]
+        tags = set(tags) - set(tags_db)
+        if len(tags) > 0:
+            print(f"Adding new tags: {tags}")
         for tag in tags:
             payload = {
                 "tag_name": tag,
             }
-            response = requests.post(tags_endpoint, json=payload, headers=headers)
+            response = requests.post(
+                tags_endpoint, json=payload, headers=headers, verify=False
+            )
+            print(f"Status Code: {response.status_code}, Response: {response.text}")
+
             if "tag_id" not in response.json():
-                print(f"Failed to create tag: {tag}")
-                continue
+                raise ValueError("Could not create tag: {tag}.")
             response_json = response.json()
             tags_map[response_json["tag_name"]] = response_json["tag_id"]
-            print(f"Status Code: {response.status_code}, Response: {response.text}")
+        else:
+            print("No new tags to add.")
```

Comment:
Should this be after the if statement? I didn't know you can have an else statement on its own.. 

Or put the for loop under the if block?

Reply:
Right now it prints "Adding new tags {..}", adds new tags, and then prints "No new tags to add".